### ISSUE TITLE
Error building image: Authentication is required

### ISSUE THREAD

**Issue by karthimohan** [rhuss/docker-maven-plugin#92]
Hi,
I am trying to use this your plugin to fetch the docker image from private registries and i am getting the below error. Could you please help me fix this issue.
Plugin Usage in my pom
```
          <plugin>
            <groupId>org.jolokia</groupId>
            <artifactId>docker-maven-plugin</artifactId>
            <version>0.11.0</version>
            <configuration>
              <registry>example.com</registry>
              <dockerHost>http://127.0.0.1:2376</dockerHost>
              <apiVersion>v1.16</apiVersion>
              <logDate>default</logDate>
              <images>
                <image>
                  <alias>accounts-server-alias</alias>
                  <name>accounts-server-name</name>
                  <registry>example.com</registry>
                  <build>
                    <from>example.com/base:latest</from>
                    <ports>
                      <port>6621</port>
                    </ports>
                    <assembly>
                      <descriptor>assembly.xml</descriptor>
                      <basedir>/opt/</basedir>
                    </assembly>
                  </build>
                </image>
              </images>
            </configuration>
          </plugin>
```
My Settings.xml
```
        <server>
            <id>example.com</id>
            <username>xxxx</username>
            <password>xxxxx</password>
        </server>
```
[INFO] Reading assembly descriptor: /home/dennis/docker/sami-local/salamis/core/accounts/src/main/docker/assembly.xml
[INFO] Copying files to /home/dennis/docker/xxxx/xxxxx/target/docker/build/maven
[INFO] Building tar: /home/dennis/docker/xxxx/xxxxx/target/docker/tmp/docker-build.tar
[INFO] DOCKER> Creating image [accounts-server-name] "accounts-server-alias"
[INFO] DOCKER>   Pulling repository example.com/base
[ERROR] Error building image: Authentication is required.
[ERROR] Authentication is required.
[INFO] ------------------------------------------------------------------------

**Comment by karthimohan** [comment]
looks like i am having the trouble during the build phase...I am able to use the image from private repository during the run phase.

**Comment by jgangemi** [comment]
i will take a look at this - i'm trying to push to a private registry that requires no authentication which is also failing.

**Comment by jgangemi** [comment]
ok - it seems my issue is different from this one.

can you post the output of the same command using `mvn -X`?

are you able to log into the private registry using `docker login`?

**Comment by rhuss** [comment]
This should be fixed in 0.11.5-M1 (and then soon 0.11.5)
